apps/frontend/src/app/modules/shared/components/page-renderer/page-renderer.component.html

```diff
@@ -1,2 +1,2 @@
 <h1 *ngIf="title">{{ title }}</h1>
-<div class="content" [innerHTML]="content"></div>
+<article class="content" [innerHTML]="content"></article>
```

Comment:
This is not necessarily an `article`, it's only an article when it's called from the `page.component` I'd wrap the page renderer there with an `article` element

Reply:
The article tag represents a self-contained composition in a document and not necessarily just an actual article